parent/pom.xml

```diff
@@ -1272,7 +1273,7 @@ hs_err_pid*.log</jgit.ignore>
 
 		<!-- ITests profile #3 of 3: won't run if -DskipITests=true -->
 		<profile>
-			<id>download-devstudio</id>
+			<id>download-jbt</id>
```

Comment:
so if i understand correct, you also need to change every build script that uses that profile ?

Reply:
Yes: https://issues.redhat.com/browse/JBIDE-28371